lib/utils.coffee

```diff
@@ -0,0 +1,91 @@
+fs = require 'fs'
+
+module.exports =
+  isWin: () ->
+    /^win/.test process.platform
+
+  getPlatformPath: (str) ->
+    if @isWin() then str.replaceAll /\//g, '\\' else str
+
+  getSrcPath: (root) ->
+    root + @getPlatformPath('/src/')
+
+#-----------
+
+  getSfCreatingItemParams: (sfCreatingDialog, root) ->
+    params = {}
+    params.srcPath = @getSrcPath(root)
+    params.metaDataType = "Apex" + sfCreatingDialog.itemType
+    if sfCreatingDialog.itemType == "Class"
+      params.extension = "cls"
+      params.fldPath = params.srcPath + @getPlatformPath('classes/')
+      params.srcLines = ["public with sharing class " + sfCreatingDialog.apiName + " {\n    \n}"]
+      params.metaLines = [
+        "<apiVersion>" + sfCreatingDialog.apiVersion + "</apiVersion>"
+        "<status>Active</status>"
+      ]
+    else if sfCreatingDialog.itemType == "Trigger"
+      params.extension = "trigger"
+      params.fldPath = params.srcPath + @getPlatformPath('triggers/')
+      params.srcLines = [
+        "trigger " + sfCreatingDialog.apiName + " on SObject_Api_Name (before update) {"
```

Comment:
I want to add selection of sObject and checkboxes for flags (before/after, create/update/etc) in creating dialog when I will be morally ready for this